test/matches.spec.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* global it describe */
+import { matches, querySelector } from '../src/index.js'
+import { withDom } from './utils.js'
+import simpleLight1 from './fixtures/simple1/light.html'
+import simpleShadow1 from './fixtures/simple1/shadow.html'
+import assert from 'assert'
+
+describe(('element matches'), () => {
+  const matchingSelector = '.container .component span'
+
+  it('light DOM - matches', () => {
+    withDom(simpleLight1, context => {
+      const element = context.querySelector(matchingSelector)
+      assert(element.matches(matchingSelector) === true)
```

Comment:
`assert.deepStrictEqual()` is preferable here (better debug message if it fails)

Reply:
Yeah, I changed to `strictEqual`